UI/src/model/selectedLinearAsset.js

```diff
@@ -280,12 +280,14 @@
       return selection.length;
     };
 
-    this.setValue = function(value) {
+    this.setValue = function(value, silently) {
```

Comment:
I don't understand what this "silently" is can you change into another name to be more recognizable

Reply:
I think silently is a good name because you want to make changes to the selected asset without firing any events

Reply:
can be replaced to "isToFireEvents" for example??

Reply:
changed it to withoutEventTriggers